data/scripts.js

```diff
@@ -1366,7 +1376,7 @@ exports.BattleScripts = {
 				case 'bonemerang': case 'precipiceblades':
 					if (hasMove['earthquake']) rejected = true;
 					break;
-				case 'icebeam':
+				case 'icebeam': case 'iceshard':
 					if (hasMove['blizzard'] || hasMove['freezedry']) rejected = true;
```

Comment:
eh, add a separate case because this affects Abomasnow who is good with Ice Shard and Blizzard